contracts/base/RageERC4626.sol

```diff
@@ -57,11 +89,7 @@ abstract contract RageERC4626 is ERC4626Upgradeable {
 
         //Additional cap on withdraw to ensure the position closed does not breach slippage tolerance
         //In case tolerance is reached only partial withdraw is executed
-        uint256 updatedAmount = beforeWithdrawClosePosition(amount);
```

Comment:
Shares should be updated based on the amount output. It is not being done atm.

Reply:
updated